lib/releases.sh

```diff
@@ -43,9 +43,10 @@ else
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
 fi
 
-# Remove the following line when CAPI release v1.3.0 is available
+# Remove the following lines when CAPI/CAPM3 v1.3.0 releases are available
 if [ "${CAPM3RELEASEBRANCH}" == "main" ]; then
  export CAPIRELEASE="v1.3.0-beta.1"
```

Comment:
we should probably use https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.3.0-rc.0 here now